src/main/resources/messages_ru.properties

```diff
@@ -1,4 +1,13 @@
 invalid.id=Идентификатор не может быть отрицательным
+invalid.new.entity.id=Идентификатор новой сущности должен быть равен нулю
+invalid.exist.id=Идентификатор существующей сущности должен быть больше нуля
+
+
+
```

Comment:
Why do we need such empty space here?

Reply:
It`s my mistake.  I`ve already fixed it.